recipe/meta.yaml

```diff
@@ -79,11 +84,3 @@ about:
 extra:
   recipe-maintainers:
     - conda-forge/qt-main
-    - ccordoba12
-    - duncanmmacleod
-    - gillins
-    - mingwandroid
-    - msarahan
-    - ocefpaf
-    - stuarteberg
-    - Tobias-Fischer
```

Comment:
These users are all redundant

Reply:
I was removed from the team with this :-/

Reply:
You should have stayed in. you are listed as part of the qt-main group
https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/meta.yaml#L253

Reply:
Your membership is likely going to be "implicit".

Reply:
Sure. But some folks may need an explicit one for day job reasons. We need to check with the team before doing this.